iroha_data_model/src/lib.rs

```diff
@@ -45,6 +45,10 @@ pub enum Parameter {
     TransactionReceiptTime(u128),
     /// TODO: write a doc
     BlockTime(u128),
+    /// Path to the block store.
+    BlockStorePath(String),
```

Comment:
This ISI parameter sets the block store path for a specific peer, right? At least that's what I understand from the test scenario. I am not sure if the whole network should be aware of the changing of the specific peer's block store path...

Reply:
@eadventurous I think we will need to wait [RFC](https://wiki.hyperledger.org/display/iroha/Iroha+Configuration).